tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2777,3 +2777,4 @@ zmq/test_gnmi_zmq.py:
     reason: "Test is for smartswitch"
     conditions:
       - "'arista' in platform"
+      - "'isolated' in topo_name"
```

Comment:
hi @Pterosaur should we add a OR condition in this case?

Reply:
Done, Please check it again.